dist/types/api/index.d.ts

```diff
@@ -0,0 +1,18 @@
+/** @format */
```

Comment:
The dist folder shouldn't be committed. These are just generated type files for npm

Reply:
It also looks like running the command has not added the types correctly

Reply:
Have fixed this and removed the dist folder along with additional clean up